modules/tutorials/pages/getting-started.adoc

```diff
@@ -0,0 +1,141 @@
+
+
+So what would this database look like under a relational compared to a document model?
+
+=== Relational model
+
+include::partial$student-database-diagrams.adoc[tags="student-record-erd"]
+
+Nothing too dramatic: the database contains a list of students and a list of courses. Each student can enrol on any number of courses, and the record of each of the student's enrollments is stored in a separate table called `enrollment` which links to the student's record and the courses he/she is enrolling on.
+
+The `enrollment` table highlights one of the problems with the relational database model: each table is based on a fixed schema that can only support a single data object type. You cannot store, for example, the enrollment records in the same table as the student, even though it may more natural to do so; after all, an `enrollment` cannot exist without a `student`, so why not store them together?
+
+=== Document model
+
+The document model gives you the same decomposition advantages as the relational model, but additionally, it also supports complex types which allows for a much more flexible design.
+
+include::partial$student-database-diagrams.adoc[tags="student-document-database-design"]
+Here's how the student record system might look as a document database.
+
+.student record
+[source,json]
+----
+{
+  "student-id": "000001",
+  "name": "Hilary Smith",
+  "date-of-birth": "21-12-1980",
+  "enrollments": [   // <1>
+    {
+      "course-id": "ART-HISTORY-00003",    // <2>
+      "date-enrolled": "07-9-2021"
+    },
+    {
+      "course-id": "GRAPHIC-DESIGN-00001",    // <2>
+      "date-enrolled": "15-9-2021"
+    }
+  ]
+}
+----
+
+<1> The document is stored in JSON format, which allows for the storage of complex types such as arrays without decomposing them to a second table. In this case, the list of `enrollment` records is stored with the student record.
+
+<2> Each `enrollment` record holds a reference to the course it relates to.
++
+[NOTE]
+====
+It would be a very bad idea to store the course record with each student:
+
+. It would lead to massive data duplication
+. It would make it very difficult to maintain the data. If the `credit-points` for a course needed to be changed, then you would need to access every student record to make the change.
+====
+
+.art history course record
+[source, json]
+----
+{
+  "course-id": "ART-HISTORY-00003",
+  "course-name": "art history",
+  "faculty": "fine art",
+  "credit-points" : 100,
+  "type": "course"
+}
+----
+
+.graphic design course record
+[source, json]
+----
+{
+  "course-id": "GRAPHIC-DESIGN-00001",
+  "course-name": "graphic design",
+  "faculty": "media and communication",
+  "credit-points" : 200,
+  "type": "course"
+}
+----
+
+
+Couchbase uses a document model which stores each database record as a `json` document. The document can can contain both simple scalar types, and complex types such as nested records and arrays.
```

Comment:
https://docs.couchbase.com/home/contribute/word-list.html#j

Think we should use `JSON` instead of `json` where possible.

Reply:
`The document can can...`
 
 Extra `can`.